specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -9945,7 +9948,7 @@
           "Node get remote login settings": { "$ref": "./examples/NodeGetRemoteLoginSettings.json" }
         },
         "summary": "Gets the settings required for remote login to a compute node.",
-        "description": "Before you can remotely login to a node using the remote login settings, you must create a user account on the node. This API can be invoked only on pools created with the virtual machine configuration property.",
+        "description": "Before you can remotely login to a node using the remote login settings, you must create a user account on the node. This API can be invoked only on pools created with the virtual machine configuration property. For pools created with a cloud service configuration, see the GetRemoteDesktop API.",
```

Comment:
The MSDNified name will go away though in the new world order, so I think it's better to use a name that is more Swagger-like since I think that's going to make it easier to discover across the clients/generated markdown.

Ideally this would be some sort of "link"